pkg/certs/certificateAuthority.go

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache v2.0 license.
+package certs
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/sha256"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"encoding/pem"
+	"fmt"
+	"math"
+	"math/big"
+	"time"
+)
+
+var (
+	//  1.3.6.1.4.1.311.104  subtree is defined for CIA MSK8S
```

Comment:
Just out of curiosity, what does "CIA" means in this context? Is it like the name of the agency? :)

Reply:
I reused code from from cert-tattoo repo. I thought it was (Confidentiality-Integrity-Authentication).

Reply:
Sounds good! Thanks! :)